src/plots/cartesian/axis_defaults.js

```diff
@@ -91,9 +91,19 @@ module.exports = function handleAxisDefaults(containerIn, containerOut, coerce,
     if(options.automargin) coerce('automargin');
 
     if(!options.noTickson &&
-        containerOut.type === 'category' && (containerOut.ticks || containerOut.showgrid)) {
+        (containerOut.type === 'category' || containerOut.type === 'multicategory') &&
+        (containerOut.ticks || containerOut.showgrid)
+    ) {
         coerce('tickson');
```

Comment:
Should `tickson: 'boundaries'` by the default on `type: 'multicategory` axes?

Reply:
yes!

Reply:
done in https://github.com/plotly/plotly.js/pull/3300/commits/2bb1388b336058e5892b68a9a5e58b453bbb3ae9